scheduler/crates/api/src/lib.rs

```diff
@@ -60,7 +60,7 @@ impl Application {
 
     async fn configure_server(context: NettuContext) -> Result<(Server, u16), std::io::Error> {
         let port = context.config.port;
-        let address = format!("0.0.0.0:{}", port);
+        let address = format!("127.0.0.1:{}", port);
```

Comment:
Might need to change this again when deploying, but for the moment, let's listen only on localhost interface (as any other app)